closed/autoconf/custom-hook.m4

```diff
@@ -338,58 +338,58 @@ AC_DEFUN([OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU],
   esac
 ])
 
-AC_DEFUN([OPENJ9_CONFIGURE_CRIU_SUPPORT],
+AC_DEFUN([OPENJ9_CONFIGURE_CRAC_SUPPORT],
 [
-  AC_MSG_CHECKING([for CRIU support])
-  AC_ARG_ENABLE([criu-support], [AS_HELP_STRING([--enable-criu-support], [enable CRIU support @<:@disabled@:>@])])
-  OPENJ9_ENABLE_CRIU_SUPPORT=false
+  AC_MSG_CHECKING([for CRAC support])
+  AC_ARG_ENABLE([crac-support], [AS_HELP_STRING([--enable-crac-support], [enable CRAC support @<:@disabled@:>@])])
```

Comment:
Saying the default is "disabled" is misleading; perhaps "platform dependent" would be better (same for CRIU).